### ISSUE TITLE
[Question] <关于告警功能改进建议>

### ISSUE THREAD

**Issue by Kylin-Guo** [dromara/hertzbeat#1213]
### Question

在生产环境使用中，告警信息会出现以下情况：

1. 监控采集间隔时间为1小时，阈值触发次数限制为1，则触发后每小时都会发送一次告警，但某些告警无需立刻处理，例如SSL证书过期天数剩下几天时，只需每天对运维人员发送一次警告即可，这种情况就只能增加采集间隔或触发次数去避免重复发送

2. 告警恢复的信息内容无法自定义，而且内容是直接贴上阈值的判断式，非HertzBeat运维人员并不清楚

3. 在使用中出现了某些告警触发了阈值但ErrMsg内容属于正常的情况，单纯的阈值无法判断，时间静默无法排除

综上所述，建议添加以下功能：
1. 告警次数限制   2. 告警恢复信息内容自定义 3. 告警内容关键词静默

**Comment by hertzbeat** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
### Question

In the production environment, the alarm information will appear in the following situations:

1. If the monitoring collection interval is 1 hour and the number of threshold triggers is limited to 1, an alarm will be sent every hour after triggering, but some alarms do not need to be processed immediately. It is enough for the operation and maintenance personnel to send a warning once. In this case, the collection interval or the number of triggers can only be increased to avoid repeated sending

2. The content of the alarm recovery information cannot be customized, and the content is a judgment formula directly pasted with the threshold value, which is not clear to non-HertzBeat operation and maintenance personnel

3. In use, some alarms trigger the threshold but the content of ErrMsg is normal. The simple threshold cannot be judged, and the time silence cannot be ruled out.

In summary, it is recommended to add the following features:
1. Limit the number of alarms 2. Customize the content of the alarm recovery information 3. Silence the keywords of the alarm content
